po/fr.po

```diff
@@ -9032,6 +9035,342 @@ msgstr "N/B Kodak T-max 400"
+
+#: ../src/iop/channelmixerrgb.c:1478
+#, c-format
+msgid "CCT: %.0f K (daylight)"
+msgstr "CCT : %.0f K (lumière du jour)"
+
+#: ../src/iop/channelmixerrgb.c:1479
+msgid ""
+"approximated correlated color temperature.\n"
+"this illuminant can be accurately modeled by a daylight spectrum,\n"
+"so its temperature is relevant and meaningful with a D illuminant."
+msgstr ""
+"température de couleur corrélée approximative.\n"
+"cet illuminant peut être modélisé avec précision par un spectre de lumière "
+"du jour,\n"
+"donc sa température est pertinente et significative avec un illuminant D."
+
+#: ../src/iop/channelmixerrgb.c:1485
+#, c-format
+msgid "CCT: %.0f K (black body)"
+msgstr "CCT : %.0f K (corps noir)"
+
+#: ../src/iop/channelmixerrgb.c:1486
+msgid ""
+"approximated correlated color temperature.\n"
+"this illuminant can be accurately modeled by a black body spectrum,\n"
+"so its temperature is relevant and meaningful with a Planckian illuminant."
+msgstr ""
+"température de couleur corrélée approximative.\n"
+"cet illuminant peut être modélisé avec précision par un spectre de corps "
+"noir,\n"
+"donc sa température est pertinente et significative avec un illuminant "
+"plankien."
+
+#: ../src/iop/channelmixerrgb.c:1492
+#, c-format
+msgid "CCT: %.0f K (invalid)"
+msgstr "CCT : %.0f K (invalide)"
+
+#: ../src/iop/channelmixerrgb.c:1493
+msgid ""
+"approximated correlated color temperature.\n"
+"this illuminant cannot be accurately modeled by a daylight or black body "
+"spectrum,\n"
+"so its temperature is not relevant and meaningful and you need to use a "
+"custom illuminant."
+msgstr ""
+"température de couleur corrélée approximative.\n"
+"cet illuminant peut être modélisé avec précision par un spectre de lumière "
+"de jour ou de corps noir,\n"
+"donc sa température est pertinente et significative avec un illuminant "
+"personnalisé."
+
+#: ../src/iop/channelmixerrgb.c:1500
+#, c-format
+msgid "CCT: undefined"
+msgstr "CCT : non défini"
+
+#: ../src/iop/channelmixerrgb.c:1501
+msgid ""
+"the approximated correlated color temperature\n"
+"cannot be computed at all so you need to use a custom illuminant."
+msgstr ""
+"la température de couleur corrélée approximative.\n"
+"ne peut être définie donc ce que vous avez à faire est d'utiliser un "
+"illuminant personnalisé."
+
+#. Parameters did not change, assume user wants to edit auto-set params and display controls
+#: ../src/iop/channelmixerrgb.c:1530
+msgid "white balance successfuly extracted from raw image"
+msgstr "la balance couleur a été extraite de l'image raw avec succèss"
+
+#: ../src/iop/channelmixerrgb.c:1548
+msgid "no white balance was found in raw image"
+msgstr "aucune balance couleur n'a été trouvée dans l'image raw"
+
+#. We need to recompute only the thumbnail
+#: ../src/iop/channelmixerrgb.c:1559
+msgid "auto-detection of white balance started…"
+msgstr "l'auto-détection de la balance couleur a démarrée..."
```

Comment:
balance de blanc

Reply:
I'm consistent but why I have write couleur instead of blanc...